### PR TITLE
Remove cef and heka, change some cef related code to plain logging.

### PR DESCRIPTION
**Pull request by EnTeQuAk**
@andymckay @jasonthomas can you take a look at this please? Just to make sure I understood everyone correctly regarding what we can remove and what is unused. All of the cef related logs now just use the general logging pipeline.

### CODE REVIEW

**Comment by jasonthomas** [comment]
lgtm!

File: src/olympia/amo/urls.py [hunk 1]
```diff
@@ -11,7 +11,6 @@
     url('^monitor(\.json)?$', never_cache(views.monitor),
         name='amo.monitor'),
     url('^loaded$', never_cache(views.loaded), name='amo.loaded'),
-    url('^csp/report$', views.cspreport, name='amo.csp.report'),
```
Comment (andymckay) [inline src/olympia/amo/urls.py]: csp reports have a completely new end point now right?

File: src/olympia/zadmin/forms.py [hunk 2]
```diff
@@ -239,64 +239,3 @@ class CompatForm(BaseCompatForm):
...
-
-class GenerateErrorForm(happyforms.Form):
-    error = forms.ChoiceField(choices=(
-        ['zerodivisionerror', 'Zero Division Error (will email)'],
-        ['iorequesterror', 'IORequest Error (no email)'],
-        ['heka_statsd', 'Heka statsd message'],
-        ['heka_json', 'Heka JSON message'],
-        ['heka_cef', 'Heka CEF message'],
-        ['heka_sentry', 'Heka Sentry message'],
-        ['amo_cef', 'AMO CEF message']))
```
Comment (andymckay) [inline src/olympia/zadmin/forms.py]: In the past I've found this helpful, certainly the first two. However we could probably do the same from the command line and check it works.

File: src/olympia/amo/urls.py (hunk 1 above)
Comment (jasonthomas) [inline src/olympia/amo/urls.py]: Yes, /__cspreport__ via nginx.

**Comment by andymckay** [comment]
lgtm

File: src/olympia/amo/urls.py (hunk 1 above)
Comment (andymckay) [inline src/olympia/amo/urls.py]: cool, then

Comment (EnTeQuAk) [inline src/olympia/amo/urls.py]: yeah, this was from years ago apparently.

File: src/olympia/zadmin/forms.py (hunk 2 above)
Comment (EnTeQuAk) [inline src/olympia/zadmin/forms.py]: what for, e.g to test logging and sentry?

Sentry has a command line tool that generates an error, maybe we could add something like that too if there's need.

Comment (andymckay) [inline src/olympia/zadmin/forms.py]: Yeah I think was mostly around when heka and CEF was coming in, we needed it to check the error went where it was meant to go to. Remove it and lets see how things go.

**Pull request merged**